common/c_cvarlist.cpp

```diff
@@ -35,6 +35,15 @@ CVAR_RANGE(			sv_gametype, "0", "Sets the game mode, values are:\n" \
 					CVARTYPE_BYTE, CVAR_SERVERARCHIVE | CVAR_SERVERINFO | CVAR_LATCH | CVAR_NOENABLEDISABLE,
 					0.0f, 3.0f)
 
+CVAR(				sv_survival, "0", "Survival mode",
+					CVARTYPE_BOOL, CVAR_SERVERARCHIVE | CVAR_SERVERINFO | CVAR_LATCH)
+
+CVAR(				sv_survival_join_timer, "15", "Survival mode join timer",
+					CVARTYPE_FLOAT, CVAR_SERVERARCHIVE | CVAR_SERVERINFO | CVAR_NOENABLEDISABLE)
+
+CVAR(				sv_maxlives, "1", "Survival mode number of lives",
```

Comment:
Can we change the CVAR name to `sv_survival_maxlives` to keep it in the same "namespace" as the other survival CVARs?

Reply:
I'd disagree, since LMS/TLMS will also be included shortly into that.